chain/chain/src/chain.rs

```diff
@@ -52,6 +58,9 @@ impl OrphanBlockPool {
     fn add(&mut self, orphan: Orphan) {
         let height_hashes = self.height_idx.entry(orphan.block.header.height).or_insert(vec![]);
         height_hashes.push(orphan.block.hash());
+        let prev_hash_entries =
```

Comment:
Where is this used?

Reply:
right below `prev_hash_entries.push(orphan.block.hash());`?

Reply:
Oh I didn't realize that it is a reference